src/components/formfields/common/validators.js

```diff
@@ -1,5 +1,6 @@
 
-export const validateString = (value = '', minimumLength = 0, maximumLength = 20) => {
+export const validateString = (inValue = '', minimumLength = 0, maximumLength = 20) => {
+  const value = inValue.trim()
```

Comment:
This might fail actually. But its only used inplaces where the variable is first checked for truthyness. So it should be OK.